app/src/main/java/de/grobox/transportr/locations/LocationAdapter.java

```diff
@@ -218,4 +221,9 @@ void setSort(FavLocationType favLocationType) {
 		resetDropDownLocations();
 	}
 
+	void setLocationFormatChanger(TransportNetwork.LocationFormat format){
```

Comment:
Naming: You're not setting the changer, but the locationFormat here.

Reply:
Right. I will change the name